hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/HFilePreadReader.java

```diff
@@ -34,12 +34,14 @@
 public class HFilePreadReader extends HFileReaderImpl {
   private static final Logger LOG = LoggerFactory.getLogger(HFileReaderImpl.class);
 
+  private static final int WAIT_TIME_FOR_CACHE_INITIALIZATION = 120 * 60 * 1000;
```

Comment:
Could it take that long just to read the file and parse the protobuff content?

Reply:
No, actually we have seen that it takes around 4 minutes. I will reduce this wait time to 10 mins.